code/src/Components/Selects/Checkbox.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styled from "styled-components";
+
+export const Checkbox = ({ checked, id, type, color, ...props }) => {
+  return (
+    <CheckboxContainer>
+      <HiddenCheckbox checked={checked} id={id} {...props} />
+      <StyledCheckbox checked={checked} color={color} type={type}>
```

Comment:
when You have more than 2 attributes it is a good practice to spread them to separate lines.